examples/istio-based-service-mesh/observability/kiali/README.md

```diff
@@ -0,0 +1,245 @@
+    deployment.apps/productpage-v1 created
+    ```
+    ```shell
+    kubectl get pods
+    ```
+    Ensure each pod has 2/2 containers in the `Ready` state
+    ```console
+    NAME                              READY   STATUS    RESTARTS   AGE
+    details-v1-558b8b4b76-2llld       2/2     Running   0          2m41s
+    productpage-v1-6987489c74-lpkgl   2/2     Running   0          2m40s
+    ratings-v1-7dc98c7588-vzftc       2/2     Running   0          2m41s
+    reviews-v1-7f99cc4496-gdxfn       2/2     Running   0          2m41s
+    reviews-v2-7d79d5bd5d-8zzqd       2/2     Running   0          2m41s
+    reviews-v3-7dbcdcbc56-m8dph       2/2     Running   0          2m41s
+    ```
+
+* Configure ingress gateway and virtual service
+    ```shell
+    kubectl apply -f gateway.yaml
+    kubectl apply -f virtualservice.yaml
+    ```
+* Validate that the sample application is accessible.  
+    ```shell
+    # Set environment variables for external ingress host and ports:
+    export INGRESS_HOST_EXTERNAL=$(kubectl -n aks-istio-ingress get service aks-istio-ingressgateway-external -o jsonpath='{.status.loadBalancer.ingress[0].ip}')
+    export INGRESS_PORT_EXTERNAL=$(kubectl -n aks-istio-ingress get service aks-istio-ingressgateway-external -o jsonpath='{.spec.ports[?(@.name=="http2")].port}')
+    export GATEWAY_URL_EXTERNAL=$INGRESS_HOST_EXTERNAL:$INGRESS_PORT_EXTERNAL
+    ```
+    Retrieve the external address of the sample application:
+    ```shell
+    echo "http://$GATEWAY_URL_EXTERNAL/productpage"
+    ```
+    Use `curl` to confirm the sample application is accessible:
+    ```shell
+    curl -s "http://${GATEWAY_URL_EXTERNAL}/productpage" | grep -o "<title>.*</title>"
+    ```
+    The expected output is:
+    ```console
+    <title>Simple Bookstore App</title>
+    ```
+
+## Steps
+### 1. Deploy Prometheus for Kiali
+Kiali requires Prometheus to collect and query metrics from your service mesh.
+
+```shell
+# Create a namespace for Prometheus
+kubectl create namespace prometheus
+
+# Add Prometheus Helm repository
+helm repo add prometheus-community https://prometheus-community.github.io/helm-charts
+helm repo update
+
+# Install Prometheus
+helm install prometheus prometheus-community/prometheus \
+  --namespace prometheus \
+  -f prometheus-values.yaml
+
+# Verify the Prometheus pods are running
+kubectl get pods -n prometheus
+```
+
+### 2. Install Kiali
+> [!Important]
+> This guide uses anonymous authentication for simplicity. For production deployments, implement appropriate authentication methods to secure your kiali dashboard from unauthorized access.
+```shell
+# Create namespace for Kiali
+kubectl create namespace kiali-operator
+
+# Add Kiali Helm repository (requires Helm 3)
+helm repo add kiali https://kiali.org/helm-charts
+helm repo update
+
+# Install Kiali operator
+helm install \
+  --namespace kiali-operator \
+  kiali-operator kiali/kiali-operator \
+  -f kiali-values.yaml
```

Comment:
How will customers get access to this file from the docs?

Reply:
I'm saying if you're reading an article like https://learn.microsoft.com/en-us/azure/aks/istio-deploy-addon, what are the steps for the user to get access to that file?